src/pages/json-schema-viewer.js

```diff
@@ -8,7 +8,7 @@ function JsonSchemaViewer() {
 
     return (
         <Layout title="JsonSchemaViewer" noFooter={true} wrapperClassName="schema-viewer-wrapper">
-            <div style={{flex: 1}}>
+            <div style={{height: 1000}}>
```

Comment:
Does anyone know how this can be rewritten more generally? 'height: 100%' does not work.

Reply:
Fixed by @zzeekk in 5ad741e3bab3b1a718afa023e62111eadaaefba3